arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -0,0 +1,277 @@
+            extentView.layer.borderWidth = 3
+        }
+    }
+    
+    @IBOutlet private var toolBar: UIToolbar!
+    @IBOutlet private var barButtonItem: UIBarButtonItem!
+    @IBOutlet private var instructionsLabel: UILabel!
+    
+    private let featureServiceURL = URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Sync/WildfireSync/FeatureServer")
+    private var generateJob: AGSGenerateGeodatabaseJob?
+    private var syncJob: AGSSyncGeodatabaseJob?
+    private var geodatabaseSyncTask: AGSGeodatabaseSyncTask!
+    private var geodatabase: AGSGeodatabase!
+    private var selectedFeature: AGSFeature!
+    private var areaOfInterest: AGSEnvelope!
+    
+    private func extentViewFrameToEnvelope() -> AGSEnvelope {
+        let frame = mapView.convert(extentView.frame, from: view)
+        
+        // Set the lower-left coner.
+        let minPoint = mapView.screen(toLocation: frame.origin)
+        
+        // Set the upper-right corner.
+        let maxPoint = mapView.screen(toLocation: CGPoint(x: frame.maxX, y: frame.maxY))
+        
+        // Return the envenlope covering the entire extent frame.
+        return AGSEnvelope(min: minPoint, max: maxPoint)
+    }
+    
+    private func addFeatureLayers() {
+        // Iterate through the layers in the service.
+        geodatabaseSyncTask?.load { [weak self] (error) in
+            if let error = error {
+                print("Could not load feature service \(error)")
+            } else {
+                guard let self = self else {
+                    return
+                }
+                if let featureServiceInfo = self.geodatabaseSyncTask?.featureServiceInfo,
+                    let map = self.mapView.map {
+                    for index in featureServiceInfo.layerInfos.indices.reversed() {
+                        let layerInfo = featureServiceInfo.layerInfos[index]
+                        // For each layer in the serice, add a layer to the map.
+                        let layerURL = self.featureServiceURL?.appendingPathComponent(String(index))
+                        let featureTable = AGSServiceFeatureTable(url: layerURL!)
+                        let featureLayer = AGSFeatureLayer(featureTable: featureTable)
+                        featureLayer.name = layerInfo.name
+                        map.operationalLayers.add(featureLayer)
+                    }
+                }
+            }
+        }
+    }
+    
+    // Clears selection in all layers of the map.
+    private func clearSelection() {
+        for layer in mapView.map!.operationalLayers {
+            if let layer = layer as? AGSFeatureLayer {
+                layer.clearSelection()
+            }
+        }
+    }
+    
+    private func didMove() {
+        self.selectedFeature = nil
+        self.barButtonItem.isEnabled = true
+        self.barButtonItem.title = "Sync geodatabase"
+        self.instructionsLabel.text = String("Tap the sync button")
```

Comment:
You do not need the explicit `String()` constructor here. Just setting the label to "Tap the sync button" as a literal will work.